content/200-concepts/100-components/02-prisma-client/260-native-upserts.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: 'Native upserts'
+metaTitle: 'Native upserts'
+metaDescription: 'How Prisma Client handles native inserts for increased speed and reliability.'
+tocDepth: 1
+---
+
+<TopBlock>
+
+A native upsert happens when Prisma Client hands over an `upsert` query to the database, instead of handling the upsert itself.
+
+Native upserts have the following advantages:
+
+- They are faster than regular upserts
+- [Unique key constraint errors](/reference/api-reference/prisma-client-reference/#unique-key-constraint-errors-on-upserts) cannot happen
+
+Prisma Client initiates a native upsert automatically when [specific criteria](#criteria-for-native-upserts) are met.
+
+To initiate a native upsert, Prisma Client sends the SQL construction [`INSERT ... ON CONFLICT SET .. WHERE`](www.prisma.io/dataguide/postgresql/inserting-and-modifying-data/insert-on-conflict) to the database.
+
+</TopBlock>
+
+## Criteria for native upserts
+
+### Prerequisites
+
+Prisma Client can initiate native upserts if your stack meets the following criteria:
+
+- You use Prisma version 4.6.0 or later
+- Your application uses a CockroachDB, PostgreSQL, or SQLite database
+
+### <inlinecode>upsert</inlinecode> query criteria
+
+Prisma Client initiates a native upsert if your `upsert` query meets the following criteria:
+
+- There are no nested queries in the `upsert`'s `create` and `update` [options](/reference/api-reference/prisma-client-reference/#options-7)
+- The query updates only one model
+- There is only one unique field in the `upsert`'s `where` option
+- The unique field in the `where` option and the unique field in the `create` option have the same value
+
+If your query does not meet these criteria, then Prisma Client handles the upsert itself, and does not initiate a native upsert.
```

Comment:
I would change this to:

If your query does not meet these criteria, then Prisma Client handles the upsert itself.